test_vm/vendor/cookbooks/dbfit_test/metadata.rb

```diff
@@ -7,7 +7,7 @@
 version          '0.3.0'
 depends          'mysql2_chef_gem'
 depends          'yum-mysql-community'
-depends          'mysql', '~> 8.2'
+depends          'mysql'
```

Comment:
Ah, after reading the docs I thought this meant any `8.2.x` but not `8.3.0` or later.